doc/nrf/protocols/bt/bt_qualification/index.rst

```diff
@@ -20,10 +20,12 @@ Pre-qualified design
 ********************
 
 A *design* is a specific configuration of hardware or software implementation (or both) of adopted Bluetooth specifications, either in parts or as a whole.
-If relevant, you can use an existing design that has already been qualified (called *pre-qualified design*).
+If relevant, you can use an existing design that has already been qualified (called *pre-qualified design*) and has a Qualification Design ID (QDID) number.
 This allows you to speed up your product's qualification process by omitting a testing phase.
 
 The |NCS| contains qualified portions of various Bluetooth features (for example, :ref:`ug_ble_controller_softdevice` versions).
+If your product uses a qualified Bluetooth stack provided as part of the |NCS|, you can inherit this qualification from Nordic Semiconductor, provided that you do not introduce any changes to the stack.
```

Comment:
"not introduce any changes to the stack": Could we add some words here about subsetting? One change that is allowed is to remove features, but that should then be reflected in the end product listing.

@omkar3141 : Can you suggest something? I don't have the detailed knowledge myself.